config/initializers/request_forgery_protection.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Enable origin-checking CSRF mitigation.
+Rails.application.config.action_controller
+  .forgery_protection_origin_check = true
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align `.forgery_protection_origin_check` with `.application` on line 4.